src/foremast/elb/format_listeners.py

```diff
@@ -159,7 +160,7 @@ def format_cert_name(env='', account='', certificate=None):
     return cert_name
 
 
-def generate_custom_cert_name(env='', account='', certificate=None):
+def generate_custom_cert_name(env='', region='', account='', certificate=None):
     """Generate a custom TLS Cert name based on a template.
 
     Args:
```

Comment:
Missing docstring for `account' parameter.

Reply:
I think you meant region, but fixed.

Reply:
Yeah, `region`. sorry.